spec/jobs/education_form/create_daily_spool_files_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe EducationForm::CreateDailySpoolFiles, type: :model, form: :education_benefits do
+  subject { described_class.new }
+
+  let(:application) do
+    {
+      fullName: {
+        first: "Mark",
+        last: "Olson"
+      }
+    }
+  end
+
+  context "#format_application" do
+    it "uses conformant sample data in the tests" do
+      expect(application).to match_vets_schema("edu-benefits-schema")
+    end
+    # TODO: Does it make sense to check against a known-good submission? Probably.
+    it "formats a 22-1990 submission in textual form" do
+      result = subject.format_application(form: "CH33_30", fullName: { first: "Mark", last: "Olson" })
+      expect(result).to include("*INIT*\nMARK\n\nOLSON")
+      expect(result).to include("Name:   Mark Olson")
```

Comment:
if we don't do a full diff here, would be good to test that right partial was rendered based on the form param `CH33_30`